WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -613,6 +621,13 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
         if (!mIsNewPost) {
+            // if we are opening an existing Post, and it contains a Story block, pre-fetch the media in case
```

Comment:
I would want to move this logic and the `fetchMediaList` function into the class we could use to consolidate all the dispatcher calls and events etc. LMK 😄

Reply:
I understand the motivations as said above - however having moved the essential logic of loading from prefs in https://github.com/wordpress-mobile/WordPress-Android/commit/0f4d87c25bde9da952d91fa9a5902a189acc0ed2, I'd argue there's little to nothing to gain from moving this call to a new class, for which also its results we don't care about listening to anyway (see how the [FluxC event listener is empty and only contains a comment](https://github.com/wordpress-mobile/WordPress-Android/pull/12939/files#diff-38b6aea9cfff5f3e0e4b7389b44de602R3123-R3127), and we could even completely remove if we wanted to). Let me know if you had something different in mind, I may be misunderstanding. 🙏

Reply:
Oh yes, I totally agree with this. After looking at 0f4d87c I understand your motivations and this can remain.